migrations/sql/V2020.03.17.16.19__update_bond_structure.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE bond ADD COLUMN institution_name varchar,
+ADD COLUMN institution_street varchar,
+ADD COLUMN institution_city varchar,
+ADD COLUMN institution_province varchar,
+ADD COLUMN institution_postal_code varchar,
```

Comment:
Given that these are similar fields to the attributes on a party, can institution be a party of type company?

Reply:
We had a larger discussion about this - you must have missed it. We had two options here 1) create a special type of party for institutions or 2) add this information on the bond object. 

Given that institutions aren't always the same - the chances of them getting re-used sounded slim - so we opted for option 2.

Reasoning, If I have 10 bonds saved to the institution BMO, all with different branches and addresses, in our parties table we would have 10 BMO party names, with 10 different addresses. 

How we surface parties doesn't allow users to see the difference between the 10 BMO options, and it didn't sound like the reuse was there, so we'd be cluttering our parties table with little benefit. 
The lower cost option was to add this data on the bond object.